packages/purpose-process/test/integration/createPurpose.test.ts

```diff
@@ -107,7 +107,9 @@ describe("createPurpose", () => {
     );
     await writeInReadmodel(toReadModelEService(eService1), eservices);
 
-    const { purpose, isRiskAnalysisValid } = await purposeService.createPurpose(
+    const {
```

Comment:
For this and other tests, let's modify the test to test the entire returned object, including metadata (see what was done in #1848 or #1832)